pages/common/cargo-clippy.md

```diff
@@ -21,12 +21,12 @@
 
 - Treat warnings as errors:
 
-`RUSTFLAGS="-Dwarnings" cargo clippy -- -D warnings`
+`cargo clippy -- --deny warnings`
 
 - Run checks and ignore warnings:
 
-`cargo clippy -- -A warnings`
+`cargo clippy -- --allow warnings`
 
-- Apply Clippy suggestion automatically (experimental and only supported on the nightly channel):
+- Apply suggestions automatically:
```

Comment:
I think this is better
```suggestion
- Apply Clippy suggestions automatically:
```
what do you think?

Reply:
👌